tests/vm/centos-8-aarch64.ks

```diff
@@ -0,0 +1,50 @@
+#version=RHEL8
```

Comment:
You may want to add description for this file
What it is used for, etc

Reply:
Good point.  Already updated this one earlier today, will update PR soon.